advanced/Scripts/webpage.sh

```diff
@@ -617,6 +639,15 @@ Teleporter() {
     php "${webroot}/admin/scripts/pi-hole/php/teleporter.php" > "${filename}"
 }
 
+Restore() {
+    if [[ "${#args[@]}" -gt 3 ]]; then
+        # webroot is sourced from basic-install above
+        php "${webroot}/admin/scripts/pi-hole/php/teleporter.php" ${args[@]:3}
```

Comment:
Stickler is complaining:

```suggestion
        php "${webroot}/admin/scripts/pi-hole/php/teleporter.php" "${args[@]:3}"
```